blog/storage.md

```diff
@@ -0,0 +1,390 @@
+title: SuperMQ Storage
```

Comment:
Add metadata something similar to https://github.com/absmach/magistrala-docs/blob/main/blog/oauth/oauth.md?plain=1#L1C1-L7C4